stylesheets/_layout.settings.scss

```diff
@@ -0,0 +1,439 @@
+    float: right;
+    width: 4em;
+}
+
+
+/**
+ * Form groups with labels adjacent to their controls
+ */
+
+.tab__settings .form__group:not(.form__group--flush):not(.form__group--toggle) > .control__label {
+    padding-right: $padding-small-horizontal * 2;
+    width: 40%;
+}
+
+.tab__settings .form__group:not(.form__group--flush):not(.form__group--toggle) .controls {
+    width: 60%;
+}
+
+
+/**
+ * Form groups with labels on top
+ */
+
+.tab__settings .form__group--top .control__label {
+    display: block;
+    margin-bottom: $padding-base-vertical;
+    padding-right: 0;
+    padding-top: 0;
+    width: 100%;
+}
+
+.tab__settings .form__group--top .controls::before {
+    display: none;
+}
+
+.tab__settings .form__group--top .controls > .form__control {
+    display: block;
+}
+
+.tab__settings .form__group--top .form__control:not(.checkbox):not(.radio) {
+    @include respond-min($screen-small) {
+        width: 100%;
+    }
+}
+
+
+/**
+ * Form groups without labels indented as per those with labels
+ */
+
+.tab__settings .form__group--indent {
+    margin-left: 0;
+    width: 100%;
+
+    &::before {
+        @include respond-min($screen-smaller) {
+            content: '';
+            display: table-cell;
+            width: 40%;
+        }
+    }
+}
+
+.tab__settings .form__group--indent .controls {
+    display: table-cell;
+}
+
+
+/**
+ * Form group labels padding adjustments for smaller views
+ */
+
+.tab__settings .form__group:not(.form__group--top):not(.form__group--toggle) > .control__label {
+    @include respond-max($screen-smaller) {
+        padding: 0 0 $padding-base-vertical;
+    }
+
+    @include respond-min-max($screen-smaller, $screen-medium) {
+        padding-right: $padding-small-horizontal;
+        width: 35%;
+    }
+}
+
+
+/**
+ * Form control file uploads alignment improvements
+ */
+
+.tab__settings .form__control.file {
+    height: $line-height-large;
+    padding-top: 14px;
+    width: 100%;
+
+    @include respond-min($screen-small) {
+        max-width: 100%;
+        width: $controls-max-width;
+    }
+}
+
+
+/**
+ * Guidance popover icon background colour should match settings column background colour
+ */
+
+.tab__settings .input-group-guidance {
+    background-color: color(gray, off-white);
+}
+
+
+/**
+ * Content area
+ */
+
+.has-settings .tab__content {
+    @include respond-min($screen-small) {
+        margin-bottom: $line-height-large;
+        padding-left: $padding-small-horizontal * 2;
+    }
+
+    @include respond-min($screen-medium) {
+        float: right;
+        max-width: calc(99% - 370px);
+        width: 74%;
+    }
+
+    .lt-ie9 & {
+        width: 64%;
+    }
+}
+
+
+/**
+ * Content header
+ */
+
+.content-header {
+    @include clear-fix(true);
+
+    border-bottom: 1px solid color(gray, lighter);
+    padding-top: $padding-base-vertical * 2;
+
+    .is-fixed & {
+        background-color: color(white);
+        position: fixed;
+        top: 0;
+        z-index: 101;
```

Comment:
Recommend adding a new var to the z-index variables stack in `_config.variables.scss`, it's been extremely handy to have a single reference of indexes when debugging z-index issues in the past